filament/src/fg/fg/RenderTargetResource.h

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (C) 2019 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef TNT_FILAMENT_FG_RENDERTARGETRESOURCE_H
+#define TNT_FILAMENT_FG_RENDERTARGETRESOURCE_H
+
+#include <backend/DriverEnums.h>
+#include <backend/TargetBufferInfo.h>
+
+#include "fg/FrameGraph.h"
+
+#include "Resource.h"
+#include "fg/ResourceAllocator.h"
+#include "ResourceNode.h"
+#include "VirtualResource.h"
+
+namespace filament {
+
+class FrameGraph;
+
+namespace fg {
+
+struct RenderTargetResource final : public VirtualResource {  // 104
```

Comment:
104?

Reply:
oh yeah, it's a remainder for me of the size of this object